src/main/java/mesosphere/marathon/client/model/v2/App.java

```diff
@@ -9,25 +9,26 @@
 import mesosphere.marathon.client.utils.ModelUtils;
 
 public class App {
-	private String id;
-	private String cmd;
-	private String version;
-	private Integer instances;
-	private Double cpus;
-	private Double mem;
-	private Collection<String> uris;
-	private List<List<String>> constraints;
-	private Container container;
-	private Map<String, String> env;
-	private Map<String, String> labels;
-	private String executor;
-	private List<Integer> ports;
-	private Collection<Task> tasks;
-	private Integer tasksStaged;
-	private Integer tasksRunning;
+    private String id;
```

Comment:
@seh Sorry about that.  I thought I took care of that with a reformat of the file, but it seems as if four spaces and tabs are used inconsistently in some files in your master branch.  I cleaned them up as best I could so it should be clear what changed now.